lib/ansible/modules/cloud/amazon/iam_cert.py

```diff
@@ -118,60 +126,67 @@ def boto_exception(err):
 
     return error
 
+
 def cert_meta(iam, name):
-    opath       = iam.get_server_certificate(name).get_server_certificate_result.\
+    opath = iam.get_server_certificate(name).get_server_certificate_result.\
```

Comment:
We're making a lot of unnecessary API calls and having a lot of horribly long lines - we can cure both at once:

```
    certificate =  iam.get_server_certificate(name).get_server_certificate_result.server_certificate
    ocert = certificate.certificate_body
    ocert_id = certificate.server_certificate_metadata.server_certificate_id
    ...
```